scripts/run_mcmc.py

```diff
@@ -268,27 +211,48 @@ def get_str_timedelta(td: timedelta) -> str:
     return str_td
 
 
+def mcmc_run_id_from_params(params: argparse.Namespace) -> McmcRunId:
+    """Make MCMC run id from command line parameters."""
+    # make MCMC run id
+    # get tree id from filepath excluding json extension
+    tree_path = Path(params.init_tree_fp)
+    tree_id = tree_path.stem
+    init_tree_id = TreeId.from_str(tree_id)
+    # ge the config id
+    # read json from file
+    with open(params.config_fp, "r") as f:
+        json_config = json.load(f)
+    config_id = McmcConfig(**json_config)
+    # get data id
+    # get data id from filepath excluding json extension
+    data_path = Path(params.data_fp)
+    data_id = data_path.stem
+    try:
+        data_id = CellSimulationId.from_str(data_id)
+    except AssertionError:
+        data_id = MutationDataId(data_id)
+    # make MC run id
+    mc_run_id = McmcRunId(params.seed, data_id, init_tree_id, config_id)
```

Comment:
Note - all namings should be able to handle `mutation data` of arbitrary names :) - if we ever switch to real data.

Reply:
Good!